src/api/app/views/webui/request/_set_bugowner_request_dialog.html.erb

```diff
@@ -5,7 +5,7 @@
         <%= form_tag({:action => "set_bugowner_request"}, :name => 'bugowner') do %>
           <p>
             <%= label_tag(:user, "User:") %><br/>
-            <input id="userbkp" name="user" type="hidden" value="False" />
```

Comment:
in the commit message:

`user` and `group` are `False` by default, so this request will
always fail because it will try to **find** a group with name `False` or
a user with login `False`.

And why? it was not assigned if it was `False` :confused:

Reply:
Commit rewritten. :hankey: :hankey: :hankey: